gc/base/TLHAllocationSupport.cpp

```diff
@@ -106,18 +109,12 @@ MM_TLHAllocationSupport::clear(MM_EnvironmentBase *env)
  * @param shouldFlush determines whether the existing cache information should be flushed back to the heap.
```

Comment:
remove this param

and while touching this file, please remove all the comments from this file to hpp